src/track/trackmetadatataglib.cpp

```diff
@@ -822,6 +822,7 @@ bool readAudioProperties(
         TrackMetadata* pTrackMetadata,
         const TagLib::File& file) {
     if (!file.isValid()) {
+        kLogger.warning() << "Cannot read audio properties from invalid file:" << file.name();
```

Comment:
Can we be sure the file is invalid at this point? I.e. could there be another situation that brings us to this point? If so, perhaps the wording would be better as "Unable to read audio properties from file: <name>. Suspect file corruption."

Reply:
The file is either inaccessible, or unreadable, or invalid. TagLib decided that it is not able to read from this file, we don't know the actual reason.

Replacing "invalid" by "inaccessible/unreadable/invalid" should cover all possible cases in a generic way. I don't want to guess about the reason without more information.